Cython/Utility/TypeConversion.c

```diff
@@ -806,23 +806,23 @@ static PyObject* __Pyx_PyUnicode_FromOrdinal_Padded(int value, Py_ssize_t ulengt
 
         char *cpos = chars + sizeof(chars);
         if (value < 0x800) {
-            *--cpos = (char) (0b10000000 | (value & 0x3f));
```

Comment:
GCC `-pedantic` warns about binary constants being C23 or GCC extension, so I decided to go old-fashion.